pom.xml

```diff
@@ -10,7 +10,7 @@
     <description>CollectionExerciseService</description>
 
     <properties>
-        <springboot.version>2.6.6</springboot.version>
+        <springboot.version>2.7.0</springboot.version>
```

Comment:
There are later semantic bug fixes associated with 2.7

Reply:
Updated the spring boot version to a later version.